frontend/src/hooks/queryBuilder/useTagValidation.ts

```diff
@@ -0,0 +1,36 @@
+import { QUERY_BUILDER_SEARCH_VALUES } from 'constants/queryBuilder';
+import { useMemo } from 'react';
+
+import { checkStringEndWIthSpace } from '../../utils/checkStringEndWIthSpace';
+import { useIsValidTag } from './useIsValidTag';
+import { useOperatorType } from './useOperatorType';
+
+type HookReturnValues = {
```

Comment:
would be good but open to suggestion 

```suggestion
type ITagValidation = {
```